packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5425,8 +5439,16 @@ export default class Meeting extends StatelessWebexPlugin {
           connectionType,
           isMultistream: this.isMultistream,
         });
+        // send media-engine.ready event at the end of a successful Meeting.addMedia()
+        NewMetrics.submitClientEvent({
+          name: 'media-engine.ready',
+          options: {
+            meetingId: this.id,
+          },
+        });
       })
       .catch((error) => {
+        // send media-engine.ready with an error - UCF uses errorCode 2004
```

Comment:
yes, but not sure if we should send it in the case if we've already sent the 'client.ice.end' from line 5409

Reply:
hmm, I think we should not as we are sending client.ice.end

Reply:
yeah, i think it's fine if we don't